src/main/java/org/tdl/vireo/cli/Cli.java

```diff
@@ -126,16 +126,32 @@ public void run(String... arg0) throws Exception {
                     }
                     for (int i = 0; i < acct; i++) {
                         String enc_pwd = passwordEncoder.encode("password");
-                        User submitter = userRepo.create("anonymous" + (i + 1) + "@example.com", "anonymous", "example", enc_pwd, Role.ROLE_ANONYMOUS);
-                        userRepo.saveAndFlush(submitter);
-                        submitter = userRepo.create("student" + (i + 1) + "@example.com", "student", "example", enc_pwd, Role.ROLE_STUDENT);
-                        userRepo.saveAndFlush(submitter);
-                        submitter = userRepo.create("reviewer" + (i + 1) + "@example.com", "reviewer", "example", enc_pwd, Role.ROLE_REVIEWER);
-                        userRepo.saveAndFlush(submitter);
-                        submitter = userRepo.create("manager" + (i + 1) + "@example.com", "", "manager", enc_pwd, Role.ROLE_MANAGER);
-                        userRepo.saveAndFlush(submitter);
-                        submitter = userRepo.create("admin" + (i + 1) + "@example.com", "", "admin", enc_pwd, Role.ROLE_ADMIN);
-                        userRepo.saveAndFlush(submitter);
+                        User testacct = userRepo.create("student" + (i + 1) + "@example.com", "student"+(i+1), "example", enc_pwd, Role.ROLE_STUDENT);
+                        System.out.println("Creating account with email "+testacct.getEmail()+" with role ROLE_STUDENT");
```

Comment:
We should be using the [logger](https://github.com/TexasDigitalLibrary/Vireo/blob/master/src/main/java/org/tdl/vireo/controller/DegreeController.java#L30) instead of System.out calls.